### PR TITLE
Changing to_uint64() and to_int64() to to_ulong() and to_long()

### PR DESCRIPTION
**Pull request by pablodiego**

### CODE REVIEW

**Comment by lucasccordeiro** [comment]
@pablodiego: There exists one failing test in this PR. Could you please take a look at it before we start reviewing this PR?

File: src/big-int/bigint.hh [hunk 1]
```diff
@@ -196,8 +196,8 @@ public:
   //operator ullong_t() const _fast;
   llong_t to_long() const _fast;
   ullong_t to_ulong() const _fast;
-  uint64_t to_uint64() const _fast;
-  int64_t to_int64() const _fast;
+  // uint64_t to_uint64() const _fast;
+  // int64_t to_int64() const _fast;
```
Comment (mikhailramalho) [inline src/big-int/bigint.hh]: Remove commented code.

File: src/solvers/z3/z3_conv.cpp [hunk 2]
```diff
@@ -25,6 +25,9 @@
 #include <util/type_byte_size.h>
 #include <z3_conv.h>
 
+typedef BigInt::llong_t llong_t;
+typedef BigInt::ullong_t ullong_t;
+
```
Comment (mikhailramalho) [inline src/solvers/z3/z3_conv.cpp]: Why do we need this?

File: src/goto-symex/builtin_functions.cpp [hunk 3]
```diff
@@ -878,7 +878,7 @@ void goto_symext::intrinsic_memset(
 
   if(can_construct)
   {
-    //uint64_t set_sz = to_constant_int2t(size).value.to_uint64();
+    //uint64_t set_sz = to_constant_int2t(size).value.to_ulong();
```
Comment (mikhailramalho) [inline src/goto-symex/builtin_functions.cpp]: Remove commented code.

File: src/solvers/z3/z3_conv.cpp (hunk 2 above)
Comment (pablodiego) [reply src/solvers/z3/z3_conv.cpp]: I forgot remove  this code.

**Comment by lucasccordeiro** [comment]
This PR is not valid anymore.

**Pull request closed without merging**